Automattic/newspack-plugin
# Muriel-styled Card component.

### All Submissions:






### Changes proposed in this Pull Request:



Closes #17.

### How to test the changes in this Pull Request:

1. Build the components: `npm ci && npm run build:webpack`
2. In a test environment running the Newspack plugin, navigate to Newspack->Subscriptions.
3. Observe two Cards, each with a single Checkbox in it.

### Some questions for discussion

1. The max width of this component is set to 484px to match the widths in the Figma mockups. Is it appropriate for the Card to determine its own width? Or, should we be placing Cards in a container that defines the width? If the former, we might need to introduce a prop to control the width of the Card (e.g. `cardWidth=small|medium|large`).
2. The Header/Subheader described [here](https://github.com/Automattic/newspack-plugin/issues/18) could be worked into the Card component as parameters? 
3. In the Figma document, some of the Card examples have top and bottom padding of 24px, while others which contain only a single element have top/bottom padding of 17px. We may need to control this via a prop as well, unless we can come up with logic to set this programmatically.

### Other information:
